src/target/tx/devo/emu_devof7/Makefile

```diff
@@ -2,5 +2,11 @@ SCREENSIZE  := text
 NUM_MODELS  ?= 10
 FILESYSTEMS := common text
 
+TARGET = emu_devof7
+FAMILY = devo
 
-include target/drivers/mcu/emu/Makefile.inc
+SRC += $(SDIR)/target/tx/devo/devof7/ia9211_map.c
+CFLAGS +=-I$(SDIR)/target/drivers/display/spi/
```

Comment:
This is weird to me.  the ia9211_map file is emu only but lives in the devof7 dir?

Reply:
no. it is a include file used by devof7 lcd driver as well.